WikiWatershed/model-my-watershed
# Cache geoprocessing service results

When the modifications for a scenario don't change, but the inputs do, we now use the cached results from the geoprocessing endpoint that generates the landcover census.

See commit messages for details.

**To Test:**
- Run `./scripts/manage migrate`, as well as `./scripts/npm install`.
- Regenerate the vendor bundle.
- Create a new project. The initial model run should take about 2 seconds.
- Change the precipitation level. The model run should finish in less than a second.
- Add a modification. This model run should take about 2 seconds.
- Change the precipitation level. The model run should finish in less than a second.
- Run the tests. Tests were added on both the front-end and back-end.

Connects to #512
